test/unit/Escrow.t.sol

```diff
@@ -571,15 +712,28 @@ contract EscrowUnitTests is UnitTest {
         hints[0] = 1;
         hints[1] = 1;
 
-        responses[0] = 1 ether;
-        responses[1] = 1 ether;
+        responses[0] = _stETH.getPooledEthByShares(1 ether);
+        responses[1] = _stETH.getPooledEthByShares(1 ether);
 
         _withdrawalQueue.setClaimableEtherResult(responses);
         vm.expectCall(
             address(_withdrawalQueue), abi.encodeCall(IWithdrawalQueue.getClaimableEther, (unstethIds, hints))
         );
 
+        assertTrue(_escrow.getEscrowState() == EscrowState.SignallingEscrow);
+        assertEq(_escrow.getRageQuitSupport().toUint256(), 0);
+
         _escrow.markUnstETHFinalized(unstethIds, hints);
```

Comment:
It would also be nice to test events here, as it should contain the correct number of finalized shares